src/rdma.c

```diff
@@ -0,0 +1,1947 @@
+/* ==========================================================================
+ * rdma.c - support RDMA protocol for transport layer.
+ * --------------------------------------------------------------------------
+ * Copyright (C) 2021-2024  zhenwei pi <pizhenwei@bytedance.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
Any reason you used the MIT license for this? The rest of Valkey is BSD-3-Clause.

I'm fine with MIT, but then we should mention it where we list 3rd party licenses and attributions: https://github.com/valkey-io/valkey-doc/blob/main/topics/license.md#third-party-files-and-licenses.

Reply:
> Any reason you used the MIT license for this? The rest of Valkey is BSD-3-Clause.
> 
> I'm fine with MIT, but then we should mention it where we list 3rd party licenses and attributions: https://github.com/valkey-io/valkey-doc/blob/main/topics/license.md#third-party-files-and-licenses.

Sure. I'll change it to BSD in the next version.